index.js

```diff
@@ -56,17 +56,19 @@ module.exports = function (namespace) {
 			return subLogger[n];
 		},
 		info: log.bind(log, 'info', 'log'),
-		json: function () {
+		json: function() {
 			for (let i = 0; i < arguments.length; i++) {
 				log("info", "log", JSON.stringify(arguments[i], null, 2));
 			}
 		},
 		time: log.bind(log, 'time', 'time'),
 		timeEnd: log.bind(log, 'time', 'timeEnd'),
 		log: log.bind(log, 'info', 'log'),
-		debug: log.bind(log, 'debug', 'log'),
+		debug: log.bind(log, 'debug', 'debug'),
+		trace: log.bind(log, 'trace', 'trace'),
```

Comment:
@czirker Do we really want to bind to `console.trace`? That will print a stack trace every time it is called, which may not be what we want.